packages/grafana-i18n/src/rules/no-translation-top-level/no-translation-top-level.test.js

```diff
@@ -14,10 +16,7 @@ RuleTester.setDefaultConfig({
   },
 });
 
-const expectedErrorMessage = 'Do not use the t() function outside of a component or function';
-
-const ruleTester = new RuleTester();
-
+// @ts-ignore
```

Comment:
ts-ignore because of a long nasty error about the rule context/config options that I couldn't quickly unravel - if anyone is able to fix it then please feel free 🙏 Same for the other rule

Reply:
i don't see this error locally btw - were you seeing it in CI or local? 🤔